STYLEGUIDE.md

```diff
@@ -27,7 +27,7 @@ The CSS does the styling.
 - Blockquotes should be used when quoting another source. Blockquotes require 4
 spaces.
 
-```none
+```nolang
```

Comment:
Actually, this is kinda a workaround, maybe don't write it down. :)

So... Herman uses https://highlightjs.org/usage/ for syntax highlighting. Any text here is interpreted as the language, and added as a class to the `code` block. If we leave it blank, right now we have highlightjs setup to autodetect the language, which causes weird highlighting here. So adding anything here that's *not* a recognizable language adds it as a class and highlightjs ignores it. But probably better to use `nohighlight` here, since that's what highlightjs recommends in their docs.

Where's that :embarrassed: emoji again?